docs/cni/kubernetes/AWS.md

```diff
@@ -176,7 +176,12 @@ aws ec2 run-instances \
 #  Include the subnet param above if using a non-default VPC
 ```
 
-Deploy at least one worker node using the following command:
+Now, deploy at least one worker node.
```

Comment:
Image ID for HVM on us-west-2 is no longer ami-99bfada9.  Recommend you simply get the user to look up the right one.